test.js

```diff
@@ -3,16 +3,7 @@
 import nock from "nock";
 import test from "ava";
 
-import Webflow, { WebflowError } from "./dist";
-
-test("Requires an access token", (t) => {
```

Comment:
@nickfla1 if we're killing the test for token on initialize, should we add a test to ensure it's present on a request?